products/filemaker.md

```diff
@@ -34,10 +34,40 @@ releases:
     eol: 2025-04-30
     link: https://help.claris.com/en/server-release-notes/content/index.html
 
+-   releaseCycle: "19.6"
+    releaseDate: 2022-12-31
+    eol: 2024-12-31 
+    link: https://support.claris.com/s/answerview?anum=000036025
+
+-   releaseCycle: "19.5"
+    releaseDate: 2022-06-31
```

Comment:
invalid date

Reply:
seems easy didn't see that coming thank you :)